doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -89,11 +89,21 @@ Matter
 Matter fork
 +++++++++++
 
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.1.0.1`` tag.
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.2.0.1`` tag.
 
 The following list summarizes the most important changes inherited from the upstream Matter:
 
-|no_changes_yet_note|
+* Added:
+
+   * Support for the Intermittently Connected Devices Management cluster.
+   * The :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_INTERVAL`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_INTERVAL`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD` and :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to manage ICD configuration.
+
+* Updated:
```

Comment:
Please check if any of the renamed Kconfig options were used in our documentation. I checked `CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT`, and it at least was used in `matter/getting_started/low_powe_configuration.rst` and `release-notes-2.1.0.rst`. For the mention in `release-notes-2.1.0.rst`, change the formatting from `:kconfig:option:` to double-backticks instead of renaming.

Additionally, these should probably be mentioned as required changes in `releases_and_maturity/migration/migration_guide_2.6.rst`.

Reply:
We have a separate task to align the documentation, as there are some extensions required: KRKNWK-17983, nevertheless you are right. If I started to touch the documentation I should have done it better :) I at least aligned the names following your recommendation and I also added entry in the migration guide.